### PR TITLE
Use GMPWraper/scripts for scripts

### PR DESCRIPTION
**Pull request by vvolkl**
BEGINRELEASENOTES
- Use standard Gaudi locations for executables

ENDRELEASENOTES

### CODE REVIEW

**Comment by andresailer** [comment]
runit.py is the example steering file, not a script to execute

**Comment by fdplacido** [comment]
> runit.py is the example steering file, not a script to execute

I still view that as part of the scripts, but it should have its own folder as it defines the algorithms to run. Something like `scripts/definitions` folder.

**Review by fdplacido — DISMISSED** [review]
Looks good to me.

**Comment by andresailer** [comment]
based on https://github.com/key4hep/k4-spack/pull/54#discussion_r490124463 we want to put runit.py in `share`

**Comment by andresailer** [comment]
Please rebase and flatten the history.

File: CMakeLists.txt [hunk 1]
```diff
@@ -47,13 +47,6 @@ include(cmake/gaudi_binary_tag.cmake)
 
 find_package(GaudiProject)
 
-configure_file(cmake/gmprun.in ${CMAKE_BINARY_DIR}/gmprun)
```
Comment (andresailer) [inline CMakeLists.txt:50]: Then you can also remove the file `cmake/gmprun.in`, right?

Comment (vvolkl) [reply CMakeLists.txt:50]: True, good catch

**Comment by fdplacido** [comment]
I fixed an issue with the installation directory.

File: GMPWrapper/CMakeLists.txt [hunk 2]
```diff
@@ -25,6 +25,8 @@ gaudi_subdir(GMPWrapper v1r0)
 # APS not sure when that future is
 gaudi_depends_on_subdirs(GaudiAlg GaudiKernel)
 gaudi_install_python_modules()
+gaudi_install_scripts()
+install(DIRECTORY ${CMAKE_CURRENT_LIST_DIR}/examples DESTINATION ${CMAKE_PROJECT_NAME})
```
Comment (andresailer) [inline GMPWrapper/CMakeLists.txt:29]: just GMPWrapper or share/GMPWrapper a.k.a. `${CMAKE_INSTALL_DATADIR}/${CMAKE_PROJECT_NAME}`?

Comment (vvolkl) [reply GMPWrapper/CMakeLists.txt:29]: @fdplacido you removed `CMAKE_INSTALL_DATADIR` in the last commit - was that on purpose?

Comment (fdplacido) [reply GMPWrapper/CMakeLists.txt:29]: Yes sorry, this was failing to install in my tests, should this go back?

**Comment by andresailer** [comment]
This is all good now?

**Review by andresailer — APPROVED** [review]

**Review by fdplacido — APPROVED** [review]
I updated the README to match the changes. All worked for me.

**Pull request merged**